scarlet/display.py

```diff
@@ -1,6 +1,7 @@
 import numpy as np
 from astropy.visualization.lupton_rgb import LinearMapping, AsinhMapping
 import matplotlib.pyplot as plt
+from matplotlib.patches import Rectangle, Polygon
 from .component import ComponentTree
 from .observation import convolve
 
```

Comment:
That depends on how large an image you create. If you do `figsize=(20,10)`, it will be rescaled to fit into the browser, and then the fonts are too small. We should rather make the sizes accessible as keywords.

Reply:
either..., or...
But we should have one of these to make plots that we could use directly in communications or papers.

Reply:
OK, different issue